### PR TITLE
Create purchase

### PR DESCRIPTION
**Pull request by Matthew-Abebe**
Screenshot (676)

### CODE REVIEW

**Comment by Matthew-Abebe** [comment]
Screenshot (669)

**Comment by Matthew-Abebe** [comment]
Screenshot (674)

**Comment by Matthew-Abebe** [comment]
Screenshot (675)

File: server/public/styles.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/* html,
+body {
+  height: 100%;
+} */
```
Comment (thebearingedge) [inline server/public/styles.css:4]: Please remove this

File: server/public/styles.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+.add-purchase-header {
+  margin-bottom: 20px;
+}
+
+.purchase-input-group {
+  position: absolute;
```
Comment (thebearingedge) [inline server/public/styles.css:11]: You should be able to do this without translate. Please review the CSS grid classes exercise

File: server/public/styles.css [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  margin-bottom: 20px;
+}
+
+.btn {
+  cursor: pointer;
+}
+
+@media only screen and (max-width: 805px) {
+  .purchase-input-group {
+    width: 350px;
```
Comment (thebearingedge) [inline server/public/styles.css:38]: This hard coded width shouldn't be necessary, check out the responsive layout exercise

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**